iot-e2e-tests/common/src/test/java/tests/integration/com/microsoft/azure/sdk/iot/iothub/serviceclient/ServiceClientTests.java

```diff
@@ -65,7 +65,8 @@
 {
     protected static String iotHubConnectionString = "";
     protected static String invalidCertificateServerConnectionString = "";
-    private static final String content = "abcdefghijklmnopqrstuvwxyz1234567890";
+    private static final byte[] SMALL_PAYLOAD = new byte[1024];
+    private static final byte[] LARGEST_PAYLOAD = new byte[65000]; // IoT Hub allows a max of 65 kb per message
```

Comment:
This is less than 65 * 1024 which I saw for the WebSockets. Should we test the max ?

Reply:
The 65 * 1204 you saw is the max AMQP frame size, and a frame consists of a payload and a handful of other properties and flags. As a result, we can't use a payload of 65 * 1024 kb, but we can get close to that limit